parachain/runtime/local/src/lib.rs

```diff
@@ -441,12 +441,14 @@ impl verifier_lightclient::Config for Runtime {
 
 parameter_types! {
 	pub const CommitInterval: BlockNumber = 5;
+	pub const MaxMessagesPerCommit: usize = 100;
```

Comment:
I'd be conservative and lower the limit to 20. Producing the commitment is very intensive (lots of hashing) and may consume the weight for the entire block.

Reply:
same for concerns on the ethereum side - all messages need to be able to be processed within 1 etherum block. 20 is probably fine.